src/components/EmptyComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import {
+   Typography,
+   Button,
+   CardMedia,
+   CardContent,
+   styled,
+   Box,
+} from '@mui/material'
+
+import Empty from '../assets/images/Empty.png'
+
+export const EmptyComponent = () => {
+   return (
+      <Container>
+         <Typography variant="h6">Список желаний</Typography>
```

Comment:
could you do this component reusable
these titles in Typography and in CardContent are static 
In the figma titles are different